MorkoUnityProject/Assets/Scripts/LocalController.cs

```diff
@@ -13,31 +13,66 @@ public class LocalController
 		private Vector3 lastMousePosition = Input.mousePosition;
 		private LayerMask groundMask = 1 << 9;
 		
-		// mouseControl = True == control with mouse
-		// mouseControl = False == control with joystick
-		bool mouseControl = true;
+		// mouseRotate = True == rotate with mouse
+		// mouseRotate = False == rotate with joystick
+		// mouseMove = True == move with KB
+		// mouseMove = False == move with GP
+		bool mouseRotate = true;
+		bool keyboardMove = true;
 		
 		private PlayerSettings playerSettings;
-		private float movementSpeed;
+		private float walkSpeed;
 		private float sneakSpeed;
 		private float runSpeed;
+		private float sideMultiplier;
+		private float backwardMultiplier;
+		private float sideRunMultiplier;
+		private float backwardRunMultiplier;
+		private float accelerationWalk;
+		private float accelerateSneak;
+		private float accelerationRun;
+		private float decelerationWalk;
+		private float decelerationRun;
 		private bool isMorko = false;
```

Comment:
Change player setting only contain one set of values, etiher player or mörkö. 

```
PlayerSettings normalSettings;
PlayerSettings morkoSettings;
PlayerSettings currentSettings => isMorko ? morkoSettings : normalSettings;
```

Change 'walkSpeed' and others to properties e.g. like this
```
private float walkSpeed => currentSettings.walkSpeed;
```

This way, if we change settings, there are fewer places in code we need to change too.